README.md

```diff
@@ -3,11 +3,35 @@
 A key-value store for Slack
 
 ### Setup
-To use, add an outgoing webhook in Slack integrations pointing to `slack-slam.herokuapp.com/message` and make the trigger word `slam`
+
+#### Install app
+Clone this repo to your computer and deploy it as you would a Rails app.
+
+If you're using Heroku, Scalingo or such, change `config/database.yml`
+to uncomment the lines with `DATABASE_URL` and comment the other `production:` vars.
+Set your own `SECRET_KEY_BASE`.
+
```

Comment:
I think the instructions you have added are great!
However I would prefer that the instructions for setting up the repo to contribute be a separate section from how to set up the application to use on Slack (which is sort of just the instructions we had before).